onnxruntime/core/providers/cpu/ml/tree_ensemble_aggregator.h

```diff
@@ -13,8 +13,13 @@ namespace ml {
 namespace detail {
 
 struct TreeNodeElementId {
-  int tree_id;
-  int node_id;
+  int64_t tree_id;
+  int64_t node_id;
+  TreeNodeElementId() {}
```

Comment:
>) {} [](http://example.com/codeflow?start=20&length=4)

What is the use case for this and does it require any initialization?

Reply:
I removed them. I changed another line calling the second constructor.